cmd/microcloudd/main.go

```diff
@@ -160,6 +161,7 @@ func main() {
 	app.PersistentFlags().BoolVarP(&daemonCmd.global.flagLogVerbose, "verbose", "v", false, "Show all information messages")
 
 	app.PersistentFlags().StringVar(&daemonCmd.flagMicroCloudDir, "state-dir", "", "Path to store state information for MicroCloud"+"``")
+	app.PersistentFlags().DurationVar(&daemonCmd.flagHeartbeatInterval, "heartbeat", time.Second*10, "Time between attempted heartbeats")
```

Comment:
Worth indicating in the comment that the default is 10s?

Reply:
Cobra adds it for us :)
```
MicroCloud daemon

Usage:
  microcloudd [flags]

Flags:
  -d, --debug                Show all debug messages
      --heartbeat duration   Time between attempted heartbeats (default 10s)
```

Reply:
Ah true, thanks!